### DISCUSSION TITLE
Mileage by driver

### DISCUSSION THREAD

**Discussion by jlestel** [teslamate-org/teslamate#3446] [Q&A] [answered]
Hi,

It seems to me that the car detects who the driver is for each drive. 
I would like to make a dashboard to show the mileage per driver.

Do you know if this information is recorded by TeslaMate? Or if it is available in the streaming data or the API?

**Comment by cwanja** [comment, marked as answer]
Been asked a few times before, but as far as the last time I checked 'driver' is not a data point the API sends.

**Reply by JakobLichterfeld** [reply]
Linking #3297
